docs/admin/kubeadm-init.md

```diff
@@ -0,0 +1,425 @@
+Allowed customization are:
+
+* provide an alternative `imageRepository` to be used instead of
+  `gcr.io/google_containers`
+* provide an `unifiedControlPlaneImage` to be used instead of different images for each control plane component
+  for control plane components
+* provide a specific `etcd.image` to be used instead of the image available at`gcr.io/google_containers`
+
+
+### Using custom certificates {#custom-certificates}
+
+By default kubeadm will generate all the certificates needed for a cluster to run.
+You can override this behaviour by providing your own certificates.
+
+To do so, you must place them in whatever directory is specified by the
+`--cert-dir` flag or `CertificatesDir` configuration file key. By default this
+is `/etc/kubernetes/pki`.
+
+If a given certificate and private key pair both exist, kubeadm will skip the
+generation step and those files will be validated and used for the prescribed
+use-case. This means you can, for example, copy an existing CA into `/etc/kubernetes/pki/ca.crt`
+and `/etc/kubernetes/pki/ca.key` and kubeadm will use this CA for signing the rest 
+of the certs.
+
+It is also possible to provide just the `ca.crt` file and not the 
+`ca.key` file (this is only available for the root CA file, not other cert pairs). 
+If all other certificates and kubeconfig files already are in place kubeadm
+recognize this condition and activates the so called "ExternalCA" mode, which also
+implies the csrsignercontroller in controller-manager won't be started.
+
+### Managing the kubeadm drop-in file for the kubelet {#kubelet-drop-in}
+
+The kubeadm deb package ships with configuration for how the kubelet should
+be run. Note that the `kubeadm` CLI command will never touch this drop-in file.
+This drop-in file belongs to the kubeadm deb/rpm package.
+
+This is what it looks like:
+
+
+```
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf"
+Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true"
+Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
+Environment="KUBELET_DNS_ARGS=--cluster-dns=10.96.0.10 --cluster-domain=cluster.local"
+Environment="KUBELET_AUTHZ_ARGS=--authorization-mode=Webhook --client-ca-file=/etc/kubernetes/pki/ca.crt"
+Environment="KUBELET_CADVISOR_ARGS=--cadvisor-port=0"
+Environment="KUBELET_CERTIFICATE_ARGS=--rotate-certificates=true --cert-dir=/var/lib/kubelet/pki"
+ExecStart=
+ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_SYSTEM_PODS_ARGS $KUBELET_NETWORK_ARGS $KUBELET_DNS_ARGS $KUBELET_AUTHZ_ARGS $KUBELET_CADVISOR_ARGS $KUBELET_CERTIFICATE_ARGS $KUBELET_EXTRA_ARGS
+```
+
+A breakdown of what/why:
+
+* `--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf` path to a kubeconfig 
```

Comment:
I think these bullet points should be treated as sentences.

*  `xxx` is a path ... specified by `--kubeconfig`.
*  `xxx` points to the ...